space-framework/pkg/msclientlib/controller.go

```diff
@@ -40,26 +39,24 @@ const (
 )
 
 type controller struct {
-	context          context.Context
-	logger           logr.Logger
-	queue            workqueue.RateLimitingInterface
-	spaceInformer    cache.SharedIndexInformer
-	multiSpaceClient *multiSpaceClient
+	context       context.Context
+	logger        logr.Logger
+	msClient      *multiSpaceClient
```

Comment:
Nothing sets this field.

Corolary: the testing is inadequate.

Reply:
Fixed.  Code was probably dropped during the rebase